<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>457</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cliente &lt;Rosa Parra&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Servidor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Servidor</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="IP_Label">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IP"/>
      </item>
      <item>
       <widget class="QLabel" name="Puerto_Label">
        <property name="text">
         <string>Puerto:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Puerto"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Conectar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Conectar</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Estudiante">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Estudiante</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Nombre_Label">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Nombre"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Correo_Label">
        <property name="text">
         <string>Correo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Correo"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="Nombre_Label_2">
        <property name="text">
         <string>Contrase√±a</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Contrasena"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="EnviarNombre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Archivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="Buscar">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AbrirArchivo"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="EnviaArchivo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Estado">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado: Desconectado</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Cliente &lt;Rosa Parra&gt;</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
